packages/financial-templates-lib/src/price-feed/DefaultPriceFeedConfigs.js

```diff
@@ -479,6 +479,13 @@ const defaultConfigs = {
     hourlyLookback: 604800,
     minTimeBetweenUpdates: 60
   },
+  uSTONKS_APR21: {
+    type: "uniswap",
+    uniswapAddress: "0xedf187890af846bd59f560827ebd2091c49b75df",
+    twapLength: 7200,
+    poolDecimals: 6,
```

Comment:
You should be able to drop this now, as it's auto-detected, based on the decimals of the constituents.
```suggestion
    twapLength: 7200,
```

Reply:
Thanks! Good to know. I verified by testing locally